ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -72,6 +79,8 @@ def __init__(
         self.fast_simulation = fast_simulation
         np.random.seed(self.seed)
         tf.set_random_seed(self.seed)
+        self.sampler_manager = sampler_manager
+        self.lesson_length = lesson_length
```

Comment:
A bit uncomfortable with multiple things called `lesson_length`. This lesson length is different than the one used by meta_curriculum, right? Is it the same as `episode_length` defined in the YAML file?

Reply:
This is the same as the episode_length in the yaml file, not sure if episode length would be appropriate either. Any thoughts about lesson_duration instead?

Reply:
Would use something that doesn't involve the word `lesson` (curriculum) or `episode` (Is the time until Done()). Maybe `resampling_interval` or something similar?

Whatever you choose, it should be the same in the yaml and in the code.